distributed/tests/test_nanny.py

```diff
@@ -26,6 +23,9 @@
 from distributed.utils import TimeoutError, parse_ports, tmpfile
 from distributed.utils_test import captured_logger, gen_cluster, gen_test, inc
 
+pytestmark = pytest.mark.gpu
+
+
 pytestmark = pytest.mark.ci1
```

Comment:
This is interesting. The second overwrites the first. Nice example why this kind of thing is not just aesthetics :)

Reply:
Oh, nice catch! After grepping through the code, it looks like this was the only place where we were overwriting `pytestmark`